imod/mf6/model_gwf.py

```diff
@@ -153,3 +166,57 @@ def update_buoyancy_package(self, transport_models_per_flow_model) -> None:
         transport_models_old = buoyancy_package.get_transport_model_names()
         if len(transport_models_old) == len(transport_models_per_flow_model):
             buoyancy_package.update_transport_models(transport_models_per_flow_model)
+
+    @classmethod
+    @standard_log_decorator()
+    def from_imod5_data(
+        cls,
+        imod5_data: dict[str, dict[str, GridDataArray]],
+        default_simulation_allocation_options: SimulationAllocationOptions,
+        default_simulation_distributing_options: SimulationDistributingOptions,
+        regridder_types: Optional[dict[str, tuple[RegridderType, str]]] = None,
+    ) -> "GroundwaterFlowModel":
```

Comment:
Please add a docstring to the method, explaining what it does

Reply:
done, added the docstring